docs/general-development/working-with-tasks-in-sharepoint-workflows-using-visual-studio-2012.md

```diff
@@ -521,7 +521,7 @@ After creating the site column, the next step is to create a specialized content
 
 <?xml version="1.0" encoding="utf-8"?>
 <Elements xmlns="http://schemas.microsoft.com/sharepoint/">
-  <!-- Parent ContentType: Workflow Task (SharePoint) (0x0108003365C4474CAE8C42BCE396314E88E51F) -->
+  <!-- Parent ContentType: Workflow Task (SharePoint 2013) (0x0108003365C4474CAE8C42BCE396314E88E51F) -->

```

Comment:
Please remove 2013 from this code listing... it's not included when creating workflows so this doesn't reflect reality.

Reply:
Hi Andrew -- thanks for the review!  I've made all requested updates, except for:

-  the change from "Workflow Task (SharePoint 2013) " to "Workflow Task (SharePoint) ", since "Workflow Task (SharePoint 2013) " does reflect reality, as seen in Figure 9 of the original article (and my development work in Visual Studio).
- "no such product "SharePoint Workflow Manager" -- I've included a link to the announcement of 'SharePoint Workflow Manager', which is the going-forward Microsoft-supported upgrade to Workflow Manager 1.0 and Service Bus...

Please let me know if additional updates will be required prior to approval - thanks!